sample-auth-common/src/main/java/com/squareup/sample/authworkflow/AuthReactor.kt

```diff
@@ -109,4 +111,15 @@ class AuthReactor(
 
   private val SubmitLogin.userInputErrorMessage: String
     get() = if (email.indexOf('@') < 0) "Invalid address" else ""
+
+  companion object {
+    /**
+     * Returns a [RunWorkflow] handle that will instruct a [WorkflowPool] to call
+     * [launch] and start a workflow.
+     */
+    fun getStarter(
+      state: AuthState = AuthState.startingState()
+    ): RunWorkflow<AuthState, AuthEvent, String> =
+      RunWorkflow(AuthReactor::class.makeWorkflowId(), state)
```

Comment:
It's generally so nice to not have to know about `ID`s anymore in client code. Could we make a utility method on `RunWorkflow` or `WorkflowHandle` to just accept a Launcher and generate the id internally?

Reply:
Will give that a shot.